package.json

```diff
@@ -6,20 +6,21 @@
   "scripts": {
     "start": "npm run storybook",
     "test": "jest",
-    "storybook": "start-storybook -p 6006 -s public",
+    "storybook": "start-storybook -p 6006",
     "build": "webpack --config webpack.config.js --mode production",
     "dev": "webpack --watch --config webpack.dev.config.js --mode development",
-    "linked": "npm run clean && mkdir -p dist && npm run cp && npm run dev && npm run cp",
+    "linked": "npm run clean && mkdir -p dist && npm run rm-react-deps && npm run cp && npm run dev",
     "buildw": "webpack --config webpack.config.js --watch",
     "build-storybook": "build-storybook -s public",
     "build-storybook-docs": "build-storybook -s public --docs",
-    "clean": "rimraf dist storybook-static test",
+    "clean": "rm -rf dist",
     "publish-ghpages": "mkdir -p build-storybook && npm run build-storybook && gh-pages -d storybook-static",
-    "cp": "cp package.json ./dist/package.json && cp README.md ./dist/README.md ",
+    "cp": "cp package.dist.json ./dist/package.json && cp README.md ./dist/README.md ",
```

Comment:
Probably worth adding to the README about what the difference is between the two package files.